### ISSUE TITLE
[DEMO] Trainable Quantum Convolution

### ISSUE THREAD

**Issue by dymat** [PennyLaneAI/qml#165]
#### General information

**Names**
Denny Mattern (denny.mattern@fokus.fraunhofer.de)
Darya Martyniuk (darya.martyniuk@fokus.fraunhofer.de)
Fabian Bergmann (fabian.bergmann@fokus.fraunhofer.de)
Henri Willems (henri.willems@fokus.fraunhofer.de)

**Affiliation**
Data Analytics Center at Fraunhofer Institute for Open Communication Systems (Fraunhofer FOKUS) [1]. This demo results from our research as part of the PlanQK consortium [2].

--------------------------------------------------------------------------------

#### Demo information

**Title**
Trainable Quantum Convolution

**Abstract**
We implement a trainable version of Quanvolutional Neural Networks [3] using parametrized RandomCircuits. Parameters are optimized using standard gradient descent. Our code is based on the "Quanvolutional Neural Networks"-Demo of Andrea Mari [4].

**Code**
 https://github.com/PlanQK/TrainableQuantumConvolution

**Relevant links**
[1] https://www.fokus.fraunhofer.de and https://www.data-analytics-center.org.

[2] https://www.planqk.de.

[3] Maxwell Henderson, Samriddhi Shakya, Shashindra Pradhan, Tristan Cook, "Quanvolutional Neural Networks: Powering Image Recognition with Quantum Circuits", 2019, arxiv:1904.04767.

[4] https://pennylane.ai/qml/demos/tutorial_quanvolution.html.

**Comment by co9olguy** [comment]
Awesome, thanks for sharing @dymat. We'll take a look at this as soon as we can and create a card for your project on the website :muscle:

**Comment by josh146** [comment]
Hi @dymat, thanks for contributing this demo! I had a look through the attached GitHub repo, the demo looks great and will make a nice addition to the PennyLane community demo page. I've begun the process of adding it to the community page in #170.
